User Story Standards

```diff
@@ -0,0 +1,238 @@
+<span style="color:#191970">**EIS-M Project User Story Standards** 
```

Comment:
Remove span tag and replace with :
`#EIS-M Project User Story Standards`

Reply:
Thanks Curtis, it's been literally years since I created a page in github, was trying to use the ADO markdown, totally forgot how to do this in Github. SO nice of you to take the time to help me figure out what I was doing wrong.